src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
Interfaces should not contain methods that are conceptually different. If there's a need for another behavior like 'draw()', it should be in a separate interface.

Reply:
I have different interfaces for calculate area and draw.